package.json

```diff
@@ -24,14 +24,15 @@
   },
   "devDependencies": {
     "assert-html-equal": "^1.0.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-register": "^6.26.0",
-    "browserify": "^17.0.0",
-    "chalk": "^5.0.1",
-    "gulp": "^4.0.2",
-    "gulp-uglify": "^3.0.2",
-    "mocha": "^10.0.0",
-    "rimraf": "^3.0.2",
+    "babel-preset-es2015": "^6.3.13",
```

Comment:
Looks like some of these were moved down to earlier versions... was that intentional?